ledger/participant-integration-api/src/main/scala/platform/store/backend/localstore/IdentityProviderCheckStorageBackend.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.daml.platform.store.backend.localstore
+
+import com.daml.ledger.api.domain.IdentityProviderId
+
+import java.sql.Connection
+
+trait IdentityProviderCheckStorageBackend {
```

Comment:
Why did you create a separate backend with just this one method?

Reply:
see PoC
UserManagementStore then later would need to check if provided identity_provider_id exists